src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,77 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int AGE_OF_TEN = 10;
+    private static final int AGE_OF_TWENTY = 20;
+    private static final String AVERAGE_LOGIN = "login12345";
+    private static final String AVERAGE_PASSWORD = "password12345";
+    private static final String SHORT_LOGIN = "ABCD";
+    private static final String SHORT_PASSWORD = "j2k";
+    private static RegistrationServiceImpl registrationService;
+
+    @BeforeAll
+    static void createReference() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void user_alreadyExist() throws RegistrationException {
+        User user = new User();
+        user.setLogin(AVERAGE_LOGIN);
+        user.setPassword(AVERAGE_PASSWORD);
+        user.setAge(AGE_OF_TWENTY);
+        registrationService.register(user);
+        for (int i = 0; i < 5; i++) {
+            assertThrows(RegistrationException.class, () -> registrationService.register(user));
+        }
+
+    }
+
+    @Test
+    void user_nullAge_notOk() {
+        User user = new User();
+        user.setAge(null);
```

Comment:
In this case all fields are `null` by default so you tests check what if all fields are null, you should have all fields filled with some dummy data, and then set a field to null, so you test what happens when all fields are correct and only one is null.